apps/debt/views/money_spent_on_room_view/money_spent_on_room_view.py

```
@@ -36,11 +37,41 @@ def total_money_spent(self) -> QuerySet[dict[str, object]]:
+            .annotate(
+                debitor_name=F("debitor__name"),
+                currency_sign=F("currency__sign"),
+                total_open_debt=Sum("value"),
+            )
+            .order_by("debitor__name")
+        )
+
+    @context
+    @property
+    def max_open_debt_per_currency(self) -> dict[str, object]:
+        """Max open debt value per currency – used for bar scaling in the template."""
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)